clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/BasicAmazonDeployDescription.groovy

```diff
@@ -80,5 +81,7 @@ class BasicAmazonDeployDescription extends AbstractAmazonCredentialsDescription
     String region
     String asgName
     Boolean useSourceCapacity
+
+    LaunchConfiguration launchConfiguration
```

Comment:
we already pull the source launch config (I think) during deploy, so we should be able to get at it without attaching it in here I think